### DISCUSSION TITLE
Neverball 1.4.0 issue with textures

### DISCUSSION THREAD

**Discussion by DonAlgo** [Neverball/neverball#306] [Q&A]
![image](https://user-images.githubusercontent.com/82674433/195933740-854aae84-073b-43cb-98d8-a1bffb62591f.png)
I downloaded Neverball 1.4.0 because I wanted to play this older version of the game, but when I opened it up, no textures loaded, not even the text, and the same happens with Neverputt.
Is there a solution?

**Comment by qwertychouskie** [comment]
It probably has to do with your OpenGL drivers being somehow broken.  What GPU do you have?  Try the latest drivers and see if it helps.

**Reply by DonAlgo** [reply]
![image](https://user-images.githubusercontent.com/82674433/195965803-0bf625b1-44f7-4b4b-9c82-a277da63e12a.png)

I dont really know about this stuff but I guess this is my gpu

**Reply by qwertychouskie** [reply]
Try this driver: https://www.intel.com/content/www/us/en/download/19344/intel-graphics-windows-dch-drivers.html?wapkw=intel%20uhd%20graphics

**Reply by DonAlgo** [reply]
I did and nothing changned
